src/app/utils/usePostDocument.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+function usePostDocument() {
```

Comment:
I like this, but `usePostDocument` is not used in your project. Maybe you like to use it in `AddDocumentForm`?

Reply:
Mhh ?